lti_consumer/lti_1p3/consumer.py

```diff
@@ -843,6 +854,9 @@ def generate_launch_request(
 
         self.set_extra_claim(proctoring_claims)
 
+        if self.proctoring_data.get("assessment_control_url"):
```

Comment:
Nit. Should we ensure that `launch_data.message_type == "LtiStartProctoring"` here as well? I think these claims are only valid in the `LtiStartProctoring` message, and we, theoretically, shouldn't send these claims in the `LtiEndAssessment` message. Or maybe this belongs in the [validate_lti_1p3_launch_data](https://github.com/openedx/xblock-lti-consumer/blob/3eaf8aa7ce0f3bd601bf318db48d7f8db50445e8/lti_consumer/api.py#L247) function or as a validator on the `Lti1p3LaunchData` class?

Reply:
ahh good call, wasn't sure if this was valid in just startProctoring or not. Lemme double check the spec b/c this is easy enough to just move inside the above codeblock